Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m

```diff
@@ -86,73 +71,73 @@
 % Add the block body to the model
 model.addBody(block)
 
-%///////////////////////////////////////
-%// DEFINE FORCES ACTING ON THE MODEL //
-%///////////////////////////////////////
-
-% Set muscle parameters
+%% Define Muscles in the Model
+% Define parameters for a Muscle
 maxIsometricForce  = 1000.0;
 optimalFiberLength = 0.25;
 tendonSlackLength  = 0.1;
 pennationAngle 	   = 0.0;
 
-% Create new muscles
+% Instantiate a Muscle
 muscle1 = Thelen2003Muscle();
 muscle1.setName('muscle1')
 muscle1.setMaxIsometricForce(maxIsometricForce)
 muscle1.setOptimalFiberLength(optimalFiberLength)
 muscle1.setTendonSlackLength(tendonSlackLength);
 muscle1.setPennationAngleAtOptimalFiberLength(pennationAngle)
 
-% Path for muscle 1
-muscle1.addNewPathPoint('muscle1-point1', ground, ArrayDouble.createVec3([0.0,0.05,-0.35]))
-muscle1.addNewPathPoint('muscle1-point2', block, ArrayDouble.createVec3([0.0,0.0,-0.05]))
+% Add Path points to muscle 1
+muscle1.addNewPathPoint('muscle1-point1', ground, Vec3(0.0,0.05,-0.35))
+muscle1.addNewPathPoint('muscle1-point2', block, Vec3(0.0,0.0,-0.05))
 
-% Repeat for Muscle 2
+% Instantiate a second Muscle
 muscle2 = Thelen2003Muscle();
 muscle2.setName('muscle2');
 muscle2.setMaxIsometricForce(maxIsometricForce)
 muscle2.setOptimalFiberLength(optimalFiberLength)
 muscle2.setTendonSlackLength(tendonSlackLength)
 muscle2.setPennationAngleAtOptimalFiberLength(pennationAngle)
 
-% Path for muscle 2
-muscle2.addNewPathPoint('muscle2-point1', ground, ArrayDouble.createVec3([0.0,0.05,0.35]))
-muscle2.addNewPathPoint('muscle2-point2', block, ArrayDouble.createVec3([0.0,0.0,0.05]))
+% Add Path points to  muscle 2
+muscle2.addNewPathPoint('muscle2-point1', ground, Vec3(0.0,0.05,0.35))
+muscle2.addNewPathPoint('muscle2-point2', block, Vec3(0.0,0.0,0.05))
 
 % Add the two muscles (as forces) to the model
 model.addForce(muscle1)
 model.addForce(muscle2);
 
-%Set up Controller
+%% Define a Controller to the Model
 initialTime = 0.0;
 finalTime = 3.0;
 
+% Instantiate a Prescribed Controller
 muscleController = PrescribedController();
 muscleController.setName('LinearRamp_Controller')
 muscleController.setActuators(model.updActuators())
 
-% Define linear functions for the control values for the two muscles
+% Define linear functions for the control values
 slopeAndIntercept1=ArrayDouble(0.0, 2);
 slopeAndIntercept2=ArrayDouble(0.0, 2);
 
-% Muscle1 control has slope of -1 starting 1 at t = 0
+% Set Muscle1 to have a control slope of -1 starting 1 at t = 0
 slopeAndIntercept1.setitem(0, -1.0/(finalTime-initialTime));
 slopeAndIntercept1.setitem(1,  1.0);
 
-% Muscle2 control has slope of 0.95 starting 0.05 at t = 0
+% Set Muscle2 to have a control slope of 0.95 starting 0.05 at t = 0
```

Comment:
Slope of `0.95/3`?

Reply:
Yeah, weird given the comment. I have tried to fix the coefficients to match the comment, and didn't get a compelling simulation— the block essentially stays in place. So instead, I will push changes that just 'fix' (delete) the comment.